bot-components/GitHub_mutations.ml

```diff
@@ -71,6 +71,62 @@ let reflect_pull_request_milestone ~bot_info issue_closer_info =
                 "The milestone of this issue was changed to reflect the one of \
                  the pull request that closed it." )
 
+let string_of_conclusion conclusion =
+  match conclusion with
+  | ACTION_REQUIRED ->
+      "ACTION_REQUIRED"
+  | CANCELLED ->
+      "ACTION_REQUIRED"
+  | FAILURE ->
+      "FAILURE"
+  | NEUTRAL ->
+      "NEUTRAL"
+  | SKIPPED ->
+      "SKIPPED"
+  | STALE ->
+      "STALE"
+  | SUCCESS ->
+      "SUCCESS"
+  | TIMED_OUT ->
+      "TIMED_OUT"
+
+let create_check_run ~bot_info ?conclusion ~name ~repo_id ~head_sha ~status
+    ~title ~text ~summary =
+  let conclusion =
+    match conclusion with None -> "" | Some c -> string_of_conclusion c
+  in
```

Comment:
I'm not sure I understand the logic around `conclusion`. If this is an optional argument, can't you keep it optional all the way to the GraphQL request (remove the `!`) instead of having to convert `None` to `""` (does this even work)?